modules/TestRunner/default.project.json

```
@@ -1,6 +1,7 @@
 {
     "name": "test-runner",
     "tree": {
+        "$className": "DataModel",
         "Src": {
             "$path": "src"
         }
```

I think this is the appropriate fix:
```suggestion
        "$path": "src"
```